tests/schema-mixed-multiline-text/input.yaml

```diff
@@ -0,0 +1,14 @@
+document:
+  nodes:
+    - kind: block
+      type: code_block
+      nodes:
+        - kind: block
+          type: code_line
+          nodes:
+            - kind: text
+              leaves:
+                - text: "already code"
+        - kind: text
```

Comment:
Here... I've been bitten by this more than once. Slate has a core rule that normalize mixed blocks and inlines/texts. It will keep the blocks, or the inlines/texts, depending on the kind of the first child. Here, Slate sees a block as first child, so it removes all inlines or texts.

Reply:
https://github.com/ianstormtaylor/slate/pull/1141